### PR TITLE
README, updating document

### PR DESCRIPTION
**Pull request by guzzilar**
### Objective
This pull request is updating README.md document.
You may see the changes at https://github.com/omise/omise-woocommerce/blob/update-readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,19 @@ The payment methods available for you to enable on your store are: **Alipay**, *
...
+The following is the list of payment methods that are supported in each country.
+
+#### 🇯🇵 Japan
+Credit / Debit Card
+
+#### 🇸🇬 Singapore
+Credit / Debit Card, and PayNow
+
+#### 🇹🇭 Thailand
+Alipay, Bill Payment: Tesco, Credit / Debit Card, Installment, Internet Banking, and TrueMoney Wallet.
```
Comment (keeratita) [inline README.md]: `Bill Payment: Tesco` -> `Bill Payment: Tesco Lotus` ?

Comment (danfowler) [inline README.md]:
```suggestion
Alipay, Bill Payment: Tesco Lotus, Credit / Debit Card, Installment, Internet Banking, and TrueMoney Wallet.
```

**Comment by guzzilar** [comment]
@keeratita @danfowler Thanks for the review 👍 I just pushed a new commit. Could you check a bit?

File: README.md [hunk 2]
```diff
@@ -76,11 +80,7 @@ Once the installation is done, the next thing that you are going to do is to con
 
 #### Enable payment methods
 
-> Note that Alipay, Bill Payment: Tesco, Installment, Internet Banking, and TrueMoney Wallet payment methods are only available for merchants that are registered in Thailand.
-
-The payment methods available for you to enable on your store are: **Alipay**, **Bill Payment: Tesco**, **Credit / Debit Card**, **Installment**, **Internet Banking**, and **TrueMoney Wallet**.
-
-1. At the Omise Settings page, Payment Methods section, there will be a number of available payment methods shown on the list.
+1. After setting up Omise Account, at the Omise Settings page, **"Payment Methods"** section, there will be a number of available payment methods shown on the list.
```
Comment (danfowler) [inline README.md]:
```suggestion
1. After setting up your Omise Account, there will be a number of available payment methods shown in the **"Payment Methods"** section of the Omise Settings page.
```

**Pull request merged**